server/server.go

```diff
@@ -67,6 +67,9 @@ type Config struct {
 	// Logging in implies approval.
 	SkipApprovalScreen bool
 
+	// If set, the server will use this connector to handle password grants
+	PasswordConnector string
```

Comment:
Same here, the connector should be initialized in `cmd` so this would be of type `connector.PasswordConnector`